### ISSUE TITLE
Feature request: Artist view

### ISSUE THREAD

**Issue by mpereira** [danbee/persephone#53]
Hi @danbee,

Thanks for working on Persephone. I really like the minimal look.

Do you have any plans on having an artist-based view, in addition to the album-based one? Something similar to what Spotify has. That would be a great feature (for me at least), since that's how I usually navigate when listening to music: artist -> album.

Thanks again.

**Comment by danbee** [comment]
Yes, some kind of artist view is in the plan, I'm just not sure when or what it will look like!

**Comment by mpereira** [comment]
Sweet.

For what it's worth, I'd say something like the Spotify UI would be pretty ok: a quick full text filter, a way to sort albums, and a pretty album cover view.

![image](https://user-images.githubusercontent.com/112426/77819630-9699cf80-70dc-11ea-9a20-ac2ad6615b48.png)

No pressure! I'll be keeping an eye on the project. Have a good weekend!
